implementation/test/TBTCSystemTest.js

```diff
@@ -9,30 +9,41 @@ const {BN, expectRevert} = require("@openzeppelin/test-helpers")
 const {expect} = require("chai")
 
 const TBTCSystem = contract.fromArtifact("TBTCSystem")
+const TestTBTCDepositToken = contract.fromArtifact("TestTBTCDepositToken")
 
 describe("TBTCSystem", async function() {
   let tbtcSystem
   let ecdsaKeepFactory
+  let tdt
 
   before(async () => {
-    const {tbtcSystemStub, ecdsaKeepFactoryStub} = await deployAndLinkAll(
+    const {
+      tbtcSystemStub,
+      ecdsaKeepFactoryStub,
+      tbtcDepositToken,
+    } = await deployAndLinkAll(
       [],
       // Though deployTestDeposit deploys a TBTCSystemStub for us, we want to
       // test TBTCSystem itself.
-      {TBTCSystemStub: TBTCSystem},
+      {TBTCSystemStub: TBTCSystem, tbtcDepositToken: TestTBTCDepositToken},
     )
     // Refer to this correctly throughout the rest of the test.
     tbtcSystem = tbtcSystemStub
     ecdsaKeepFactory = ecdsaKeepFactoryStub
+    tdt = tbtcDepositToken
   })
 
   describe("requestNewKeep()", async () => {
     let openKeepFee
+
     before(async () => {
       openKeepFee = await ecdsaKeepFactory.openKeepFeeEstimate.call()
+      await tdt.forceMint(accounts[0], web3.utils.toBN(accounts[0]))
     })
+
     it("sends caller as owner to open new keep", async () => {
       const expectedKeepOwner = accounts[2]
+      await tdt.forceMint(accounts[0], web3.utils.toBN(expectedKeepOwner))
```

Comment:
We already have it in `before` block, do we need to duplicate it here?

Reply:
it this specific case yes, the before block was minting the token for accounts[0], but 
 expectedKeepOwner  = accounts[2] 

This will be changed because of your comment [here](https://github.com/keep-network/tbtc/pull/514#discussion_r388831194) though